File: practice/src/selenium/SeleniumBaseTest.java
package selenium;

import org.testng.annotations.AfterSuite;
import org.testng.annotations.BeforeSuite;

public class SeleniumBaseTest {

    @BeforeSuite
    public void initWebDriver() {
        WebDriverHolder.INSTANCE.init(ChromeDriverMode.NORMAL);
    }

    @AfterSuite
    public void closeWebDriver() {
        WebDriverHolder.INSTANCE.closeDriver();
    }
}

File: practice/src/selenium/SeleniumBaseTest.kt
package selenium

import org.testng.annotations.AfterSuite
import org.testng.annotations.BeforeSuite

open class SeleniumBaseTest {

    @BeforeSuite
    fun initWebDriver() {
        WebDriverHolder.init(ChromeDriverMode.NORMAL)
    }

    @AfterSuite
    fun closeWebDriver() {
        WebDriverHolder.closeDriver()
    }
}